pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -87,6 +88,19 @@ ceph -n client.admin auth get-or-create osd."$OSD_ID" mon 'allow profile osd' mg
 if [[ "$CV_MODE" == "lvm" ]]; then
 	TMP_DIR=$(mktemp -d)
 
+	# "ceph.conf" must have the "fsid" global configuration after the following Ceph's PR is merged.
+	# https://github.com/ceph/ceph/commit/25655e5a8829e001adf467511a6bde8142b0a575
+	cp -a /etc/ceph /tmp/
+	chmod u+w /tmp/ceph/ceph.conf
+	if grep -q "^\[global]$" /tmp/ceph/ceph.conf ; then
+        sed -i "s/^\[global]/[global]\nfsid = $CEPH_FSID/" /tmp/ceph/ceph.conf
+    else
+        echo "[global]
+fsid = $CEPH_FSID" >>/tmp/ceph/ceph.conf
+    fi
+	chmod u-w /tmp/ceph/ceph.conf
+	mount --bind /tmp/ceph /etc/ceph
```

Comment:
Can we pass a different `ceph.conf` to c-v instead of this bindmount? With `CEPH_ARGS="--conf /var/lib/rook/rook-ceph/rook-ceph.config"` for instance.

Reply:
It works for non-portable OSDs. OSD prepare pods create "/var/lib/rook/rook-ceph/rook-ceph.config" but OSD pods don't. So if an OSD pod is created in a different node than where it is prepared.

We can create this file in an extra initContianer, but IMO,  it's overkill,

Reply:
NO I just meant to do `export CEPH_CONF=/tmp/ceph/ceph.conf` and remove `chmod` and `mount --bind /tmp/ceph /etc/ceph`.